gpflow/conditionals.py

```diff
@@ -142,18 +143,37 @@ def _sample_conditional(Xnew, feat, kern, f, *, full_output_cov=False, q_sqrt=No
     :return: N x P (full_output_cov = False) or N x P x P (full_output_cov = True)
     """
     logger.debug("sample conditional: InducingFeature Kernel")
-    mean, var = conditional(Xnew, feat, kern, f, full_cov=False, full_output_cov=full_output_cov,
+    mean, var = conditional(Xnew, feat, kern, f, full_cov=full_cov, full_output_cov=full_output_cov,
                             q_sqrt=q_sqrt, white=white)  # N x P, N x P (x P)
-    cov_structure = "full" if full_output_cov else "diag"
-    return _sample_mvn(mean, var, cov_structure)
+    if full_cov:
+        if full_output_cov:
+            raise NotImplementedError("The combination of both full_cov and full_output_cov is not "
+                                      "implemented for sample_conditional.")
+		N, D = tf.shape(mean)[0], tf.shape(mean)[1]
+		S = num_samples
+		jittermat = settings.numerics.jitter_level * tf.eye(N, batch_shape=[D], dtype=settings.float_type)  # [D, N, N]
+		L = tf.cholesky(var + jittermat)  # [D, N, N]
+		nu = tf.random_normal([D, N, S], dtype=float_type)  # [D, N, S]
```

Comment:
you can reuse the `_mvn_sample` code here with two transposes (on the mean). Also, it would be great if all this code was negatively indexed so supported arbitrary leading dim

Reply:
Yeah, this is by no means meant to be done! I just put together the minimal amount of code to support what we need in gpflux... it'd also be good for the sample_mvn to handle num_samples, etc. Feel free to push to this branch!